R/export.R

```diff
@@ -70,7 +70,7 @@ createTextFiles <- function(study, directoryname, calcOverlaps = FALSE) {
   dir.create(directoryname, showWarnings = FALSE, recursive = TRUE)
   exportSamples(study, directoryname)
   exportFeatures(study, directoryname)
-  exportModels(study, directoryname)
+  exportModels(study, directoryname, na="null")
```

Comment:
What is the motivation for only wanting to apply this to the "models" output and not other JSON output like "tests" and "annotations"?

Reply:
Companion tools that need to be updated in sequence. This gives us flexibility with output types, which I need (for now) internally. This change won't impact any external users of OmicNavigator. I anticipate `na=null` to become the default argument for `write_json` in time.